tests/state/_common/actions/useIndexed-test.js

```diff
@@ -0,0 +1,115 @@
+import {assert} from 'chai';
+import slash from 'slash';
+import commonActions from '../../../../src/state/_common/actions';
+import testBatchRunner from '../../helpers';
+import {commonActionTypesObj as actionTypes} from '../../../constants';
+
+const tests = [
+	{
+		name: 'It register index and loads indexed data.',
```

Comment:
What about the case when the data are already indexed?

Reply:
useIndexed only wraps "ensureIndexed" and add registering of the index. ensureIndexed-test solve case when data are already loaded.